tests/testthat/_snaps/stress_test_arguments.md

```diff
@@ -3,17 +3,16 @@
     Code
       stress_test_arguments
     Output
-      # A tibble: 10 x 6
-         name                    type      default allowed              min   max  
-         <chr>                   <chr>     <chr>   <chr>                <chr> <chr>
-       1 asset_type              character <NA>    equity, bonds, loans <NA>  <NA> 
-       2 lgd_senior_claims       double    0.45    <NA>                 0.3   0.6  
-       3 lgd_subordinated_claims double    0.75    <NA>                 0.6   0.9  
-       4 terminal_value          double    0       <NA>                 0     0.1  
-       5 risk_free_rate          double    0.02    <NA>                 0     0.05 
-       6 discount_rate           double    0.02    <NA>                 -0.01 0.05 
-       7 div_netprofit_prop_coef double    1       <NA>                 0.8   1    
-       8 shock_year              double    2030    <NA>                 2025  2035 
-       9 term                    double    2       <NA>                 1     10   
-      10 company_exclusion       logical   TRUE    TRUE, FALSE          <NA>  <NA> 
+      # A tibble: 9 x 6
+        name                    type      default allowed              min   max  
+        <chr>                   <chr>     <chr>   <chr>                <chr> <chr>
+      1 asset_type              character <NA>    equity, bonds, loans <NA>  <NA> 
+      2 lgd_senior_claims       double    0.45    <NA>                 0.3   0.6  
+      3 lgd_subordinated_claims double    0.75    <NA>                 0.6   0.9  
+      4 risk_free_rate          double    0.02    <NA>                 0     0.05 
+      5 discount_rate           double    0.02    <NA>                 -0.01 0.05 
+      6 div_netprofit_prop_coef double    1       <NA>                 0.8   1    
+      7 shock_year              double    2030    <NA>                 2025  2035 
+      8 term                    double    2       <NA>                 1     10   
+      9 company_exclusion       logical   TRUE    TRUE, FALSE          <NA>  <NA> 
```

Comment:
Oh cool, you updated the snapshot!!! Are you manually changing the `is_me()` to register yourself? Now that we can write outputs outside dropbox, or return them as a list, then I think I could adapt the tests to they automatically run for you, me, and whoever runs `devtools::test()`.

Reply:
Ah you noticed, so all good.
I did not change is_me, this snapshot test should not be wrapped in it.

Before adapting the current tests let us please talk about it. I would like to use the opportunity to automated the change_checks and extend covered cases.